### PR TITLE
add chat id, channel id to chatmessage

### PR DESCRIPTION
**Pull request by v-kanasi**
Adding chat id and channel identity to chatmessage markdown.

### CODE REVIEW

File: api-reference/beta/resources/chatmessage.md [hunk 1]
```diff
@@ -67,7 +67,8 @@ Represents an individual chat message within a [channel](channel.md) or [chat](c
 |reactions| [chatMessageReaction](./chatmessagereaction.md) collection | Reactions for this chat message (for example, Like).|
 |locale|string|Locale of the chat message set by the client.|
 | policyViolation | [chatMessagePolicyViolation](../resources/chatmessagepolicyviolation.md) |Defines the properties of a policy violation set by a data loss prevention (DLP) application.|
-
+|chat id|string|  This is the id of the chat the message belongs.|
```
Comment (AkJo) [inline api-reference/beta/resources/chatmessage.md:70]: This needs to match the name of the property. 

Please look at https://graph.microsoft.com/beta/$metadata to mirror the schema.

File: api-reference/beta/resources/chatmessage.md [hunk 2]
```diff
@@ -67,7 +67,8 @@ Represents an individual chat message within a [channel](channel.md) or [chat](c
...
+|channel identity|string|  This is the id of the channel the message was posted.|
```
Comment (AkJo) [inline api-reference/beta/resources/chatmessage.md:71]: Same here

Comment (AkJo) [inline api-reference/beta/resources/chatmessage.md:71]: This is not a string.

File: api-reference/beta/resources/chatmessage.md (hunk 1 above)
Comment (AkJo) [inline api-reference/beta/resources/chatmessage.md:70]: Identity of the chat in which the message was posted.

File: api-reference/beta/resources/chatmessage.md (hunk 2 above)
Comment (AkJo) [inline api-reference/beta/resources/chatmessage.md:71]: Identity of the channel in which the message was posted.

Comment (AkJo) [inline api-reference/beta/resources/chatmessage.md:71]: You will also need a new file channelIdentity resource.

File: api-reference/beta/resources/chatmessage.md [hunk 3]
```diff
@@ -67,7 +67,8 @@ Represents an individual chat message within a [channel](channel.md) or [chat](c
...
 ## JSON representation
```
Comment (AkJo) [inline api-reference/beta/resources/chatmessage.md:73]: You will also need to update the Teams.Core.json changelog file.

Please refer to this PR for a sample: https://github.com/microsoftgraph/microsoft-graph-docs/pull/11158

General instructions:
https://dev.azure.com/msazure/One/_wiki/wikis/Microsoft%20Graph%20Partners/103834/Changelog

File: api-reference/beta/resources/chatMessagechannelIdentity.md [hunk 4]
```diff
@@ -0,0 +1,51 @@
+---
+title: "channelIdentity resource type"
```
Comment (AkJo) [inline api-reference/beta/resources/chatMessagechannelIdentity.md:2]: The name of the file will be based the resource name->
channelIdentity.md

File: api-reference/beta/resources/chatMessagechannelIdentity.md [hunk 5]
```diff
@@ -0,0 +1,51 @@
...
+description: "Chat message surfaced two new properties ChannelIdentity and Chat identity. Channel identity has channel id and team id  ."
```
Comment (AkJo) [inline api-reference/beta/resources/chatMessagechannelIdentity.md:3]: Represents the identity of a channel in MS Teams.

File: api-reference/beta/resources/chatMessagechannelIdentity.md [hunk 6]
```diff
@@ -0,0 +1,51 @@
...
+author: "Kanaka"
+doc_type: resourcePageType
+localization_priority: Normal
+ms.prod: "microsoft-teams"
+---
+# channelidentity resource type
```
Comment (AkJo) [inline api-reference/beta/resources/chatMessagechannelIdentity.md:9]: Camel casing

File: api-reference/beta/resources/chatMessagechannelIdentity.md [hunk 7]
```diff
@@ -0,0 +1,51 @@
...
+
+[!INCLUDE [beta-disclaimer](../../includes/beta-disclaimer.md)]
+
+
+ChannelIdentity has ChannelID and teamID which tells the channel  and team where the message was posted
```
Comment (AkJo) [inline api-reference/beta/resources/chatMessagechannelIdentity.md:14]: A channelIdentity represents identificatory information about a channel in Microsoft Teams.

File: api-reference/beta/resources/chatMessagechannelIdentity.md [hunk 8]
```diff
@@ -0,0 +1,51 @@
...
+
+## Properties
+
+| Property   | Type |Description|
+|:---------------|:--------|:----------|
+|channelId|string|  This is the identity of the channel in which the message was posted.|
```
Comment (AkJo) [inline api-reference/beta/resources/chatMessagechannelIdentity.md:20]: The channel ID.

File: api-reference/beta/resources/chatmessage.md [hunk 9]
```diff
@@ -108,6 +109,8 @@ The following is a JSON representation of the resource.
   "reactions": [{"@odata.type": "microsoft.graph.chatMessageReaction"}],
   "locale": "string",
   "policyViolation": {"@odata.type": "microsoft.graph.chatMessagePolicyViolation"},
+  "chatId": "string",
+  "channelIdentity": {"@odata.type": "microsoft.graph.chatMessagechannelIdentity"},
```
Comment (AkJo) [inline api-reference/beta/resources/chatmessage.md:113]: microsoft.graph.channelIdentity

File: api-reference/beta/resources/chatmessage.md [hunk 10]
```diff
@@ -67,7 +67,8 @@ Represents an individual chat message within a [channel](channel.md) or [chat](c
 |reactions| [chatMessageReaction](./chatmessagereaction.md) collection | Reactions for this chat message (for example, Like).|
 |locale|string|Locale of the chat message set by the client.|
 | policyViolation | [chatMessagePolicyViolation](../resources/chatmessagepolicyviolation.md) |Defines the properties of a policy violation set by a data loss prevention (DLP) application.|
-
+|chatId|string|  This is the identity of the chat in which the message was posted.|
+|channelIdentity | [chatMessagechannelIdentity](../resources/chatMessagechannelIdentity.md) |This is the identity of the channel in which the message was posted.|
```
Comment (AkJo) [inline api-reference/beta/resources/chatmessage.md:71]: If populated, then this is the identity of the channel in which the message was posted.

File: api-reference/beta/resources/chatmessage.md [hunk 11]
```diff
@@ -67,7 +67,8 @@ Represents an individual chat message within a [channel](channel.md) or [chat](c
 |reactions| [chatMessageReaction](./chatmessagereaction.md) collection | Reactions for this chat message (for example, Like).|
 |locale|string|Locale of the chat message set by the client.|
 | policyViolation | [chatMessagePolicyViolation](../resources/chatmessagepolicyviolation.md) |Defines the properties of a policy violation set by a data loss prevention (DLP) application.|
-
+|chatId|string|  This is the identity of the chat in which the message was posted.|
```
Comment (AkJo) [inline api-reference/beta/resources/chatmessage.md:70]: If populated, then this is the identity of the chat in which the message was posted.

File: changelog/Microsoft.Teams.Core.json [hunk 12]
```diff
@@ -1,6 +1,33 @@
...
+             "ChangeType": "Addition",
+             "Description": "Added **chatId** property to the [chatMessage](https://docs.microsoft.com/en-us/graph/api/resources/chatmessage?view=graph-rest-beta) resource.",
+             "Target": "chatMessage"
+           },
+           {
+              "Id": "8018F283-A3C1-4C6E-B0B5-B2A0965272DF",
+             "ApiChange": "Resource",
+             "ChangedApiName": "channelIdentity",
+             "ChangeType": "Addition",
+             "Description": "Added [chatMessagechannelIdentity](https://docs.microsoft.com/en-us/graph/api/resources/chatMessagechannelIdentity?view=graph-rest-beta) resource type",
```
Comment (AkJo) [inline changelog/Microsoft.Teams.Core.json:19]: The name of the entity is 'channelIdentity' , and not chatMessageChannelIdentity.

Please fix at other places too.

Comment (AkJo) [inline changelog/Microsoft.Teams.Core.json:19]: channelIdentity

**Review by Lauragra — APPROVED** [review]

**Pull request merged**